web-registry/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -64,6 +64,7 @@ const Dashboard = (): JSX.Element => {
         label: 'Bridge',
         icon: <BridgeIcon />,
         href: '/ecocredits/bridge',
+        hidden: process.env.REACT_APP_LEDGER_CHAIN_ID === 'regen-1', // TODO: Hide in PROD - remove when Bridge is ready
       },
```

Comment:
I considered using `process.env.NODE_ENV === 'production'`, but this is the same result.